react-props/my-app/src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import { Header } from './Header';
+import { ImageContainer } from './ImageContainer';
+import { ImageCaption } from './ImageCaption';
+import { ImageDescription } from './ImageDescription';
+import { ButtonContainer } from './ButtonContainer';
+import './App.css';
+
+const headerText: string = 'React Image Bank';
+const imgSRC: string = '/starry-sky.jpeg';
```

Comment:
Although this is just stylistic, it is common to capitalize only the first letter of shortened words or acronyms. So, `imgSrc`.